app/src/lib/stores/accounts-store.ts

```diff
@@ -43,6 +43,7 @@ interface IAccount {
   readonly avatarURL: string
   readonly id: number
   readonly name: string
+  readonly scopes?: ReadonlyArray<string>
```

Comment:
Does this have to be optional? I'm thinking we can require this be set and use `[]` if we have no scopes.